lib/decompress/huf_decompress.c

```diff
@@ -404,6 +450,7 @@ static void HUF_fillDTableX4Level2(HUF_DEltX4* DTable, U32 sizeLog, const U32 co
 }
 
 typedef U32 rankVal_t[HUF_TABLELOG_MAX][HUF_TABLELOG_MAX + 1];
```

Comment:
Maybe we should redefine `rankVal_t` as a table of `rankValCol_t`.
That would make the relation more obvious.